src/lib/checkbox/checkbox.scss

```diff
@@ -240,6 +240,7 @@ md-checkbox {
   vertical-align: middle;
   white-space: nowrap;
   width: $md-checkbox-size;
+  flex: 0 0 $md-checkbox-size;
```

Comment:
@DevVersion yeah, would also work this way. I can change it very quick if you want. Sorry for that!